src/status_im/contexts/communities/actions/channel_view_details/view.cljs

```diff
@@ -6,32 +6,44 @@
             [status-im.common.contact-list-item.view :as contact-list-item]
             [status-im.common.contact-list.view :as contact-list]
             [status-im.common.home.actions.view :as home.actions]
+            [status-im.constants :as constants]
             [status-im.contexts.communities.actions.channel-view-details.style :as style]
             [status-im.feature-flags :as ff]
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
 (defn- contact-item
-  [public-key]
-  (let [show-profile-actions                         #(rf/dispatch
-                                                       [:show-bottom-sheet
-                                                        {:content (fn [] [home.actions/contact-actions
-                                                                          {:public-key public-key}])}])
-        [primary-name secondary-name]                (rf/sub [:contacts/contact-two-names-by-identity
-                                                              public-key])
-        {:keys [ens-verified added? compressed-key]} (rf/sub [:contacts/contact-by-address public-key])
-        theme                                        (quo.theme/use-theme)]
+  [public-key _ _ {:keys [theme]}]
+  (let [show-profile-actions          (rn/use-callback
```

Comment:
Some optimizations here:

1. Before this PR, every little scroll in the member list would trigger up-to 20 contact items being processed by Reagent, 2 * 10 (default initial item to render in section list) because props were different on every render of the section list item due to uncached functions. We now cache functions and Reagent reports processing only new elements on the member's list.

2. Moved the theme hook out and passed the theme as a prop.